src/main/java/seedu/address/model/timetable/exceptions/WrongTimeException.java

```diff
@@ -0,0 +1,10 @@
+package seedu.address.model.timetable.exceptions;
+
+/**
+ * Represents a timing error.
+ */
+public class WrongTimeException extends RuntimeException {
```

Comment:
This as well, cos EduMate may not recognise it for the front end.

Reply:
This will have to wait depending on how it will be integrated. For now, the logic is preliminary implemented.

Reply:
Alright I understand.